static/manifests/calico/ClusterRole/calico-kube-controllers.yaml

```diff
@@ -23,6 +25,16 @@ rules:
       - get
       - list
       - watch
+  # Services are monitored for service LoadBalancer IP allocation
```

Comment:
Do you have a reference to the upstream manifests? At first glance, I can't find this: https://github.com/projectcalico/calico/blob/v3.29.1/charts/calico/templates/calico-kube-controllers-rbac.yaml#L44-L139

Reply:
I included a 3.29.2 bugfix regarding permissions, but thinking about it, I'll close this PR and fix everything in the v3.29.2 version bump,